docs/docker-compose/README.md

```diff
@@ -296,11 +319,12 @@ net start winnat
 
 The table below shows the location of the publicly available `Dockerfile` for the containers used in the Community deployment.
 
-| Container | Dockerfile Location |
-|----------|-------------|
-| alfresco | https://github.com/Alfresco/acs-packaging/blob/master/docker-alfresco/Dockerfile |
-| share | https://github.com/Alfresco/share/blob/master/packaging/docker/Dockerfile |
-| solr6 | https://github.com/Alfresco/SearchServices/blob/master/search-services/packaging/src/docker/Dockerfile |
+| Container                       | Dockerfile Location                                                                                                                     |
+|---------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------|
+| alfresco                        | https://github.com/Alfresco/acs-packaging/blob/master/docker-alfresco/Dockerfile                                                        |
+| share                           | https://github.com/Alfresco/share/blob/master/packaging/docker/Dockerfile                                                               |
+| content-app                     | https://github.com/Alfresco/alfresco-content-app/blob/master/Dockerfile                                                         |
```

Comment:
just content-app line added here, aside from Intellij autoformatting making it prettier